src/main/drivers/pwm_output_stm32f4xx.c

```diff
@@ -82,8 +82,11 @@ void pwmWriteDigital(uint8_t index, uint16_t value)
         packet <<= 1;
     }
 
-    DMA_SetCurrDataCounter(motor->timerHardware->dmaStream, MOTOR_DMA_BUFFER_SIZE);
-    DMA_Cmd(motor->timerHardware->dmaStream, ENABLE);
+    DMA_Stream_TypeDef *stream = motor->timerHardware->dmaStream;
+    TIM_DMACmd(motor->timerHardware->tim, motor->timerDmaSource, DISABLE);
+    DMA_SetCurrDataCounter(stream, MOTOR_DMA_BUFFER_SIZE);
+    DMA_ClearITPendingBit(stream, motor->dmaFlag);
```

Comment:
Carefull here, DMA channel may be inaccesible until ALL flags are cleared (https://github.com/betaflight/betaflight/pull/1968)